src/main/java/io/specto/hoverfly/junit/core/config/HoverflyConfiguration.java

```diff
@@ -188,4 +191,19 @@ public void setPlainHttpTunneling(boolean plainHttpTunneling) {
         this.plainHttpTunneling = plainHttpTunneling;
     }
 
+    public LocalMiddleware getLocalMiddleware() {
+        return localMiddleware;
+    }
+
+    public void setLocalMiddleware(LocalMiddleware localMiddleware) {
+        this.localMiddleware = localMiddleware;
+    }
+
+    public boolean isMiddlewareEnabled() {
+        return localMiddleware != null && isNotBlank(localMiddleware.getBinary()) && isNotBlank(localMiddleware.getPath());
+    }
+
+    private boolean isNotBlank(String str) {
```

Comment:
Feel free to use`StringUtils.isNotBlank()` from Apache Common which is already in the classpath.

Reply:
There is no reason we can't remove it. For the time being, if end-user can exclude it from transitive dependency if there is a clash.